source/docs/romi-robot/hardware.rst

```diff
@@ -0,0 +1,60 @@
+Romi Hardware & Assembly
+========================
+
+.. warning:: We are working hard on getting the Romi documentation complete. The documentation here should be enough to get your robot assembled and working. Please check back often for updates.
+
+To get started with the Romi, you will need to have the necessary hardware.
+
+1. `Romi Kit from Pololu <https://www.pololu.com/product/4022>`__ – Use coupon code ``FIRSTROMI548249`` for a discount.
+2. `Raspberry Pi <https://www.amazon.com/gp/product/B07BFH96M3/>`__ – 3B+ or 4
+3. `16GB (or larger) Micro SD card <https://www.amazon.com/dp/B073K14CVB/>`__ and adapter to connect to a computer
```

Comment:
I think the WPILibPI docs said 4 gb SD card. Are those wrong, or is the Romi image that much larger?